src/nnpack/nnlib.jl

```diff
@@ -0,0 +1,71 @@
+const AA{N} = AbstractArray{Float32,N}
+const AA1 = Union{AA{2}, AA{3}, AA{4}, AA{5}}
+
+relu(x::AA1; inplace::Bool = true, nthreads = NNPACK_CPU_THREADS) =
```

Comment:
I don't think we should overload NNlib's scalar activation functions like this.

Reply:
So should I leave out the activation functions?